sass/components/_view.scss

```diff
@@ -86,6 +87,50 @@
 			}
 		}
 	}
+	.cq-view-dialog
+	{
+		margin-top: 100;
+		background: #0b1838;
+		opacity: .7;
+		margin-bottom: 5px;
+		width: 340px;
+		height: 80px !important;
+		border-radius: 5px;
+		color: white;
+		&.open{
+			position: relative;
+		}
+	}
+	.cq-view-dialog .dlg-content
+	{
+		text-align: center;
+		padding-top: 10px;
+		display: flex;
+		justify-content:center;
+		>.ic-icon
+		{
+			-moz-transition: none;
+			-webkit-transition: none;
+			-ms-transition: none;
+    		-o-transition: color 0 ease-in;
+			transition: none;
```

Comment:
You don't need to write so much stuff; use the transition mixin defined in `_variables.scss`